src/main/java/bio/terra/cli/serialization/userfacing/UFSshKey.java

```diff
@@ -0,0 +1,67 @@
+package bio.terra.cli.serialization.userfacing;
+
+import bio.terra.cli.utils.UserIO;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonPOJOBuilder;
+import java.io.PrintStream;
+
+@JsonDeserialize(builder = UFSshKey.Builder.class)
+public class UFSshKey {
```

Comment:
Maybe name this `UFSshKeyPair` since it has two keys in it. Same for business object, PD, etc.